pom.xml

```diff
@@ -204,6 +204,12 @@
     <!-- Overrides the version in the jboss-integration-platform-bom. A peer PR will be sent to jboss-integration-platform project.
          Please move this new version number there when possible -->
     <version.org.jboss.forge.roaster>2.19.4.Final</version.org.jboss.forge.roaster>
+
+    <!-- deps for incremental compilation -->
+    <version.org.apache.maven.incremental>3.3.9</version.org.apache.maven.incremental>
+    <version.plexus.interpolation>1.21</version.plexus.interpolation>
+    <version.commons.cli>1.3.1</version.commons.cli>
+    <version.plexus.logging>1.0.3</version.plexus.logging>
```

Comment:
These should already be coming from the ip-bom (https://github.com/jboss-integration/jboss-integration-platform-bom). The only one we will likely need to upgrade is the {{commons-cli}}.